OpenRA.Mods.Common/Traits/Multipliers/ProductionCostMultiplier.cs

```diff
@@ -25,11 +25,11 @@ public class ProductionCostMultiplierInfo : TraitInfo<ProductionCostMultiplier>,
 		public readonly string[] Prerequisites = { };
 
 		[Desc("Queues that this cost will apply.")]
-		public readonly HashSet<string> Queue = new HashSet<string>();
+		public readonly HashSet<string> Queues = new HashSet<string>();
```

Comment:
I suppose this needs an update rule now. :(

Reply:
Okay, but it seems this field is unused in the default mods.